roles/agent/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+- name: Install the bluechi-agent package
+  ansible.builtin.package:
+    name:
+    - bluechi-agent
+    - bluechi-selinux
+
+- name: Copy the BlueChi Agent configuration file
+  ansible.builtin.template:
+    src: agent.conf.j2
+    dest: /etc/bluechi/agent.conf.d/agent.conf
```

Comment:
Just an FYI: (feel free to close anytime)
The configuration files in `/etc/bluechi/agent.conf.d/` are applied sequentially in ascending order. So if there is another file in this directory, e.g. `test.conf`, it could override the settings. To ensure that this config is the last, it could be passed in via the `-c` cli option of BlueChi - which would require changing the service file, so probably not a good option.

Reply:
In my view, if you're using Ansible to deploy BlueChi, all the configuration should be passed during the run of the playbook. In addition, changes to the configuration should also go through another run on the playbook (with changed variables).
This does raise the need to support all configuration keys. For this I've just opened #4